go/vt/schemadiff/schema.go

```diff
@@ -263,6 +263,12 @@ func (s *Schema) normalize() error {
 	// - then we only want views that depend on 1st level views or on tables. These are 2nd level views.
 	// - etc.
 	// we stop when we have been unable to find a view in an iteration.
+
+	// it's possible that the hasn't been any table in this schema.
+	// for purposes of the algorithm, the iterationLevel must be at least 1
```

Comment:
Consistent sentence capitalization and punctuation would make this easier to read.

`It's possible that there's never been any tables in this schema.`? Is that we mean? Or `If there are currently no tables in the schema, we still need to perform one iteration run.`

Reply:
Edited the text altogether.